backend/core/models.py

```diff
@@ -436,10 +437,18 @@ def decline(self):
         self.save()
 
 
+def get_random_ical_id() -> str:
+    return get_random_string(length=48)
+
+
 class Team(CommonInfo):
     name = models.CharField(max_length=255)
     is_public = models.BooleanField(default=False)
     recipes = GenericRelation("Recipe", related_query_name="owner_team")
+    ical_id = models.TextField(
```

Comment:
should this be unique?

Reply:
There's the team id in the URL so it should be fine

However, the migration for this field needs to be updated:

```
❯ yak django sqlmigrate core 0076
BEGIN;
--
-- Add field ical_id to team
--
ALTER TABLE "core_team" ADD COLUMN "ical_id" text DEFAULT 'G6GRPshGx09IuiapcEX0L16VVRwHbIMFHhYJPu78li8En1lB' NOT NULL;
ALTER TABLE "core_team" ALTER COLUMN "ical_id" DROP DEFAULT;
COMMIT;
```